{"name":"user go to test fails at first try, then solve()","status":"passed","stage":"finished","description":"","steps":[],"attachments":[],"parameters":[],"start":1648380145003,"stop":1648380154269,"uuid":"2263810b-bdcb-4290-9110-75669163f4ed","historyId":"31b612ba417b5ef99d9ab97212666198","fullName":"edu.platform.e2e.TestSolveEndToEndTest.user go to test fails at first try, then solve","labels":[{"name":"host","value":"alexander-UX310UAR"},{"name":"thread","value":"23326@alexander-UX310UAR.Test worker(1)"},{"name":"framework","value":"junit-platform"},{"name":"language","value":"java"},{"name":"package","value":"edu.platform.e2e.TestSolveEndToEndTest"},{"name":"testClass","value":"edu.platform.e2e.TestSolveEndToEndTest"},{"name":"testMethod","value":"user go to test fails at first try, then solve"},{"name":"suite","value":"edu.platform.e2e.TestSolveEndToEndTest"}],"links":[]}